packages/office-ui-fabric-react/src/components/FocusZone/examples/FocusZone.Disabled.Example.tsx

```diff
@@ -3,31 +3,27 @@ import * as React from 'react';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import { FocusZone, FocusZoneDirection } from 'office-ui-fabric-react/lib/FocusZone';
 import { TextField } from 'office-ui-fabric-react/lib/TextField';
-import './FocusZone.Disabled.Example.scss';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
 
-export const FocusZoneDisabledExample = () => (
-  <div className="ms-FocusZoneDisabledExample">
-    <div className="ms-Row">
-      <FocusZone direction={FocusZoneDirection.horizontal}>
+export const FocusZoneDisabledExample: React.StatelessComponent = () => (
+  <Stack gap={20} horizontalAlign="start">
```

Comment:
I'm hoping we can get #8247 in sooner than later rather than continue to sprinkle `gap` in our examples which I think it a styling anti-pattern. (I'm also curious if `warnDeprecations` on `gap` will cause build issues with all of these as well.)

Reply:
Yeah @khmakoto will have to either do a find/replace of `gap` before checking in his PR. An alternative to prevent further delays to the big PR would be omitting the `warnDeprecations` call for now, then doing a separate PR to add the warning and fix all instances.

Reply:
I'm working on getting the changes so we can check that PR. Hopefully I'll get that done today.